server/plugin.go

```diff
@@ -38,7 +38,12 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 
-	if err := p.API.RegisterCommand(command.GetCommand()); err != nil {
+	comm, err := command.GetCommand(p.API)
```

Comment:
```suggestion
	cmd, err := command.GetCommand(p.API)
```
nitpicking here, but we commonly use `cmd` as shorthand for `command`

Reply:
Okay, that sounds good, I updated the PR with the mentioned change